Source/Urho3D/Math/Color.cpp

```diff
@@ -64,6 +64,14 @@ Vector3 Color::ToHSV() const
     return Vector3(h, s, v);
 }
 
+void Color::FromUInt(unsigned color)
+{
+    a_ = (float)(((color >> 24) & 0xff) / 255);

```

Comment:
color / 255 int diivision?

Reply:
Should I be converting 255 to unsigned? Or something else?

Reply:
when you divide two integer numbers, result is rounded to int
just write 255.0f

Reply:
Done in last commit.